server/mscalendar/notification.go

```diff
@@ -141,7 +141,7 @@ func (processor *notificationProcessor) processNotification(n *remote.Notificati
 	n.Subscription = sub.Remote
 	n.SubscriptionCreator = creator.Remote
 
-	client := processor.Remote.MakeClient(context.Background(), processor.Poster, processor.Store, creator.MattermostUserID, creator.OAuth2Token)
+	client := processor.Remote.MakeClient(context.Background(), creator.OAuth2Token, processor.Store, sub.MattermostCreatorID, store.GetCheckUserStatus(processor.Store, processor.Logger, sub.MattermostCreatorID), store.GetChangeUserStatus(processor.Store, processor.Logger, sub.MattermostCreatorID, processor.Poster))
```

Comment:
Can we instantiate the callbacks into local variables before this line? This line is a bit noisy and difficult to read at the moment